src/pages/workspace/expensifyCard/WorkspaceExpensifyCardBankAccounts.tsx

```diff
@@ -45,14 +44,12 @@ function WorkspaceExpensifyCardBankAccounts({route}: WorkspaceExpensifyCardBankA
             return null;
         }
 
-        // const eligibleBankAccounts = Object.values(bankAccountsList).filter((bankAccount) => bankAccount.accountData.allowDebit || bankAccount.accountData.type === CONST.BANK_ACCOUNT.TYPE.BUSINESS);
         const eligibleBankAccounts = CardUtils.getEligibleBankAccountsForCard(bankAccountsList);
 
         return eligibleBankAccounts.map((bankAccount) => {
             const bankName = (bankAccount.accountData?.addressName ?? '') as BankName;
             const bankAccountNumber = bankAccount.accountData?.accountNumber ?? '';
-            // TODO: change 1 to 0 - applied for testing purposes, as sometimes accountData lacks fundID
-            const bankAccountID = bankAccount.accountData?.fundID ?? 1;
+            const bankAccountID = bankAccount.accountData?.bankAccountID ?? 0;
```

Comment:
@koko57 
1. First question, why do we use fundID before?
2. Why do we use 0 as the default value in API params? If `bankAccount.accountData?.bankAccountID` is undefined we should prevent from calling API instead of calling API with value as 0

Reply:
@mountiny should we use fundID or bankAccountID?

Reply:
it should be bankAccountID

Reply:
@koko57 Bump

> Why do we use 0 as the default value in API params? If bankAccount.accountData?.bankAccountID is undefined we should prevent from calling API instead of calling API with value as 0

Reply:
@DylanDylann for eslint not to complain, but ok, fixed it for you